### ISSUE TITLE
RCS1085 false positive

### ISSUE THREAD

**Issue by miegir** [JosefPihrt/RoslynTools#17]
```C#
class A
{
    private readonly string s = "s";
    // RCS1085 (consider replacing property with auto-implemented property) at the next line:
    public object S => s;
}
```
Replacing with auto property can break the existing code here because the property type is more generic than the field type, and not every method that was called on the field can be called on the property.

It's very interesting though that the warning can be suppressed in this small sample (but in the real code it is not working) by adding any comment after the `;` in the property declaration (for this reason the comment in my example is above and not after the property).
